cut/copy/paste via shortcuts has been corrected. Earlier, event wasn't

shapon2004 opened the pull request:
cut/copy/paste via shortcuts has been corrected. Earlier, event wasn't fired on empty list.

shapon2004 committed:
cut/copy/paste via shortcuts has been corrected. Earlier, event wasn't
fired on empty list.

shapon2004 committed:
right-click menu, for the share option we can simply label this as
"Share" rather than "Share file or folder".

shapon2004 committed:
Fixed : currently when I select a folder from the tree and hit share, it
opens the Access rights mgt rather than the Share

shapon2004 committed:
Move file/folder by drag&drop between the portal and the tree view

shapon2004 committed:
Merge branch 'apromore:development' into development

raboczi commented on Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/common/FolderTreeRenderer.java:
```diff
@@ -127,7 +129,38 @@ public void onEvent(Event event) throws Exception {
+				FolderTreeNode dropedToItem = ((Treeitem) event.getTarget().getParent()).getValue();
+				FolderType selectedFolder = (FolderType) dropedToItem.getData();
+
+				Object dropedObject = null;
```
`dropedObject` should be `droppedObject`.

raboczi commented on Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/common/FolderTreeRenderer.java:
```diff
@@ -127,7 +129,38 @@ public void onEvent(Event event) throws Exception {
+							mainC.getPortalSession().getCurrentFolder());
+				}
+			} catch (Exception e) {
+				LOGGER.error("Error Occured in Drag and Drop on Tree", e);
```
"Occured" should be "occurred".

raboczi commented on Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/BaseListboxController.java:
```diff
@@ -621,6 +621,15 @@ public void paste() throws Exception {
     refreshContent();
   }
+  
+	public void drop(FolderType dropToFolder,Object droppObject, FolderType currentFolder) throws Exception {
```
`droppObject` should be `dropObject`.

raboczi commented on Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/renderer/SummaryItemRenderer.java:
```diff
@@ -276,6 +284,29 @@ public void onEvent(Event event) throws Exception {
+            public void onEvent(DropEvent event) throws Exception {
+            	try {
+            	Listitem droppedToItem = (Listitem)event.getTarget();
+            	Object dropedObject=null;
```
`dropedObject` should be `droppedObject`.

raboczi commented on Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/workspaceOptions/CopyAndPasteController.java:
```diff
@@ -344,4 +344,45 @@ public void paste(FolderType currentFolder) throws Exception {
+			} catch (Exception e) {
+				Messagebox.show(Labels.getLabel("portal_failedCut_message"), "Apromore", Messagebox.OK,
+						Messagebox.ERROR);
+				LOGGER.error(e.getMessage());
```
It'd be better to have a stack trace.  Perhaps `LOGGER.error("Unable to perform Cut", e);`?

raboczi commented on Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/workspaceOptions/CopyAndPasteController.java:
```diff
@@ -344,4 +344,45 @@ public void paste(FolderType currentFolder) throws Exception {
+			}
+		} catch (Exception e) {
+			Messagebox.show(Labels.getLabel("portal_failedPaste_message"), "Apromore", Messagebox.OK, Messagebox.ERROR);
+			LOGGER.error(e.getMessage());
```
Again, it'd be better to have a stack trace, e.g. `LOGGER.error("Unable to perform Paste", e);`

raboczi commented on Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/common/FolderTreeRenderer.java:
```diff
@@ -127,7 +129,38 @@ public void onEvent(Event event) throws Exception {
+		public void onEvent(DropEvent event) throws Exception {
+
+			try {
+				FolderTreeNode dropedToItem = ((Treeitem) event.getTarget().getParent()).getValue();
```
`dropedToItem` should be `droppedToItem`.   Or perhaps `dropTarget`?

raboczi reviewed (changes requested):
Minor changes requested; mainly typos and some fussiness about logging.

shapon2004 committed:
Merge branch 'apromore:development' into development

Commit:
FIxed Based on Review Feedback..

https://github.com/apromore/ApromoreCore/pull/1446/files/444466d71e20203e35409b8815098abdca867d40

shapon2004 reviewed (commented):
Done

shapon2004 reviewed (commented):
Done

raboczi reviewed (approved):
All requested changes addressed; approved.
